client/src/components/all-components/CodeBlock/StyleComponents/JSONText.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * React component for rendering JSON-formatted text with syntax highlighting.
+ *
+ * The `JSONText` component takes a single prop, `children`, which should be a string containing
+ * the JSON-formatted text to be displayed. It applies syntax highlighting to the text, differentiating
+ * between JSON data and operators, making it more readable for users.
+ *
+ * @param {Object} props - The props for the `JSONText` component.
+ * @param {string} props.children - props.children - The React element representing the content of the code line, in this
+ * case, the children will simply be passed in text, which will then be styled and syntactically highlighted similarly to JSON-formatted text.
+ *
+ * @return {JSX.Element} The rendered component that displays JSON text with syntax highlighting.
+ */
+
+const JSONText = ({ children }) => {
+  const highlightSyntax = (text) => {
+    {
+      /* Operator regex only colors operator symbols in a passed in child, such as /?+, etc */
```

Comment:
inline comments

Reply:
removed regex from each component and inserted into an export file with inline comments.